devices.go

```diff
@@ -443,9 +444,9 @@ type DeviceDeleteRequest struct {
 	Force bool `json:"force_delete"`
 }
 type DeviceReinstallFields struct {
-	OperatingSystem  string `json:"operating_system,omitempty"`
-	PreserveData     bool   `json:"preserve_data,omitempty"`
-	DeprovisionFast  bool   `json:"deprovision_fast,omitempty"`
+	OperatingSystem string `json:"operating_system,omitempty"`
```

Comment:
so sayeth `go fmt`.

Other changes with unexpected formatting (to be addressed in a separate PR):

- batches.go
- organizations_test.go
- ports.go